tests/suite-unit/PluginFlyvemdmPackage.php

```diff
@@ -37,13 +37,84 @@ class PluginFlyvemdmPackage extends CommonTestCase {
 
    public function beforeTestMethod($method) {
       switch ($method) {
+         case 'testShowForm':
+         case 'testPrepareInputForAdd':
+         case 'testAddNeededInfoToInput':
          case 'testPrepareInputForUpdate':
          case 'testPostGetFromDB':
             $this->login('glpi', 'glpi');
             break;
       }
    }
 
+   /**
+    * @param $method
+    */
+   public function afterTestMethod($method) {
+      switch ($method) {
+         case 'testShowForm':
+         case 'testPrepareInputForAdd':
+         case 'testAddNeededInfoToInput':
+         case 'testPrepareInputForUpdate':
+            parent::afterTestMethod($method);
+            \Session::destroy();
```

Comment:
Is this call really useful ? Destriying sessions is handled when we need to build a new one in the test tools package

Reply:
Is useful for ensure that we are cleaning up the session to start a new one on the next test with the same values. The CommonTestCase hasn't have a session destroy on the login action to ensure that we would have always the same session vars.

Reply:
OK, understood